pkg/machines/getOSList.py

```diff
@@ -26,6 +48,14 @@
     osObj['releaseDate'] = os.get_release_date_string() or ""
     osObj['eolDate'] = os.get_eol_date_string() or ""
     osObj['codename'] = os.get_codename() or ""
+    osObj['recommendedResources'] = {}
+    recommendedResources = os.get_recommended_resources() or []
+    for resource in list(_OsinfoIter(recommendedResources)):
```

Comment:
Does this need to be converted into list? Wouldn't the iterable just do it?

Reply:
It's not needed indeed. Removing the casting.